### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by IharDorinel**
There are changes in top section and svg-sprites.

### CODE REVIEW

File: top_section.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <label for="select_rooms"></label>
+        <select id="select_rooms" name="select">
+          <option value="1 Rooms">1 Rooms</option>
+          <option value="1">1</option>
+          <option value="2">2</option>
+          <option value="3">3</option>
+          <option value="4">4</option>
+          <option value="5">5</option>
+        </select>
+      </div>
```
Comment (romos21) [inline top_section.html:103]: @IharDorinel use type=number for input here

Comment (romos21) [reply top_section.html:103]: remove select and set input type="number" -> <input type="number"/>

File: top_section_styles.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.rooms_form {
+    width: 100px;
+    height: 65px;
+    border: 3px solid #F3F3F4;
+    background-color: #F3F3F4;
+    border-radius: 8px;
+    float: left;
+}
```
Comment (romos21) [inline top_section_styles.css:182]: there are the same forms - you can set a common class for them

File: top_section.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    <form action="/" method="get">
+      <div class="destination_form">
+
+        <input type="search" id="destination" name="destination" placeholder="New York">
+        <label for id="destination"></label>
+      </div>
+
+      <div class="dates_form">
+        <input id="dates" type="date" name="calendar" placeholder="Tue 15 Sept - Sat 19 Sept">
+        <label for id="dates"></label>
```
Comment (romos21) [inline top_section.html:67]: @IharDorinel what's role does this label play?

File: top_section.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  <div class="booking_rectangle">
+
+    <form action="/" method="get">
+      <div class="destination_form">
+
+        <input type="search" id="destination" name="destination" placeholder="New York">
+        <label for id="destination"></label>
+      </div>
+
+      <div class="dates_form">
```
Comment (romos21) [inline top_section.html:65]: @IharDorinel why do you wrap input in div here? Set styles for input

File: top_section_styles.css [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+.button {
```
Comment (romos21) [inline top_section_styles.css:184]: @IharDorinel set border color to this button

**Comment by IharDorinel** [comment]
@romos21 these comments are fixed in lesson-7

**Pull request merged**